src/Modules/Admin/Commands.Admin/help/Get-PowerBIActivityEvents.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Get-PowerBIActivityEvents
+
+## SYNOPSIS
+Retrieves the audit activity events for the tenant.
+
+## SYNTAX
+
+```
+Get-PowerBIActivityEvents -StartDateTime <String> -EndDateTime <String> [-ActivityType <String>]
+ [-ResultType <Int32>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Retrieves the audit activity events by the tenant administrator for the organization.
```

Comment:
Retrieves the audit activity events for the calling user's tenant.
Before you run this command, make sure you log in using Connect-PowerBIServiceAccount. 
This cmdlet requires the calling user to be a tenant administrator of the Power BI service.